code/modules/clothing/spacesuits/rig/rig_construction.dm

```diff
@@ -154,6 +154,15 @@
 	board_type = /obj/item/circuitboard/rig_assembly/illegal/hacker
 	target_board_type = /obj/item/circuitboard/rig_assembly/illegal/targeting/hacker
 
+/obj/item/rig_assembly/combat/illegal/hacker/machinist
+	name = "cybersuit control module assembly"
+	desc = "The assembly frame for an advanced powered armor suit with many cyberwarfare enhancements. Comes with built-in insulated gloves for safely tampering with electronics."
+	icon_base = "hacker"
+	icon_state = "hacker1"
```

Comment:
Don't need to set them, if they're the same as the parent

Reply:
Not sure what this is asking to do - do you want this removed from the file? I just did a copy and paste of the recipe.